orchestra/tests/workflows/test_dir/v1/version.json

```diff
@@ -48,6 +48,7 @@
             "name": "Step 3",
             "description": "The third step",
             "is_human": true,
+            "completion_ends_project": true,
```

Comment:
do you use this for anything?

Reply:
The answer to this question is related to the query on orchestra/tests/helpers/workflow.py . I use this to test if completion_ends_project got loaded from this workflow version in assert_test_dir_v1_loaded function.